desc/magnetic_fields/_core.py

```diff
@@ -2579,13 +2580,14 @@ def field_line_integrate(
     source_grid=None,
     rtol=1e-8,
     atol=1e-8,
-    maxstep=1000,
+    max_steps=None,
     min_step_size=1e-8,
     solver=Tsit5(),
     bounds_R=(0, np.inf),
     bounds_Z=(-np.inf, np.inf),
     chunk_size=None,
```

Comment:
This changes a functionality, chunk_size before was for the BS-law computation, now it is over the field lines. I still think one should be able to specify the bs_chunk_size